blockchains/eth/lib/withdrawals_decoder.js

```diff
@@ -0,0 +1,36 @@
+const { computeGasExpense, computeGasExpenseBase36 } = require('./util');
+const constants = require('./constants');
+
+class WithdrawalsDecoder {
+  constructor(web3, web3Wrapper) {
+    this.web3Wrapper = web3Wrapper;
+  }
+
+  getWithdrawal(withdrawal, block, blockNumber) {
+    return [{
+      from: constants.BURN_ADDRESS,
+      to: withdrawal.address,
+      value: this.web3Wrapper.parseHexToNumber(withdrawal.amount),
+      valueExactBase36: this.web3Wrapper.parseValueToBN(withdrawal.amount).toString(36),
+      blockNumber: blockNumber,
+      timestamp: this.web3Wrapper.parseHexToNumber(block.timestamp),
+      transactionHash: withdrawal.validatorIndex,
```

Comment:
This is not correct, withdrawal transfers don't have hashes so I wasn't sure what to put